### DISCUSSION TITLE
about/faq/

### DISCUSSION THREAD

**Discussion by giscus** [crossdb-org/docs#8] [Announcements]
# about/faq/

The Fastest Embedded Database in the world

https://crossdb.org/about/faq/

**Comment by kouke0305** [comment]
When will it be open source?

**Reply by jcwangxp** [reply]
The project is redesigned and rewritten from scratch, so it takes time to code and test, plan to push initial draft code in the middle of this month.

**Comment by sirinath** [comment]
MPL is not an issue but might be slightly more appealing if was ASL2.0/MIT.

**Reply by jcwangxp** [reply]
MPL is very loose already, the original license is AGPL. Contributions are welcome to make this project better.
